src/components/DirectoryCityGrid.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import { Address } from "@yext/pages/components";
+import { Address } from "@yext/sites-components";
```

Comment:
I think we did the rename to pages-components

Reply:
We did, but it hasn't released yet